// @techiediaries/ngx-qrcode/esm5/lib/qrcode.module.js
import { __decorate } from "tslib";
import { NgModule } from '@angular/core';
import { QrcodeComponent } from './qrcode.component';
import * as ɵngcc0 from '@angular/core';
var NgxQRCodeModule = /** @class */ (function () {
    function NgxQRCodeModule() {
    }
NgxQRCodeModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: NgxQRCodeModule });
NgxQRCodeModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function NgxQRCodeModule_Factory(t) { return new (t || NgxQRCodeModule)(); } });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(NgxQRCodeModule, { declarations: function () { return [QrcodeComponent]; }, exports: function () { return [QrcodeComponent]; } }); })();
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(NgxQRCodeModule, [{
        type: NgModule,
        args: [{
                declarations: [QrcodeComponent],
                exports: [QrcodeComponent]
            }]
    }], function () { return []; }, null); })();
    return NgxQRCodeModule;
}());
export { NgxQRCodeModule };
