// ngx-webcam/esm2015/src/app/modules/webcam/webcam.module.js
import { NgModule } from '@angular/core';
import { CommonModule } from '@angular/common';
import { WebcamComponent } from './webcam/webcam.component';
import * as ɵngcc0 from '@angular/core';
const COMPONENTS = [
    WebcamComponent
];
export class WebcamModule {
}
WebcamModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: WebcamModule });
WebcamModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function WebcamModule_Factory(t) { return new (t || WebcamModule)(); }, imports: [[
            CommonModule
        ]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(WebcamModule, { declarations: function () { return [WebcamComponent]; }, imports: function () { return [CommonModule]; }, exports: function () { return [WebcamComponent]; } }); })();
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(WebcamModule, [{
        type: NgModule,
        args: [{
                imports: [
                    CommonModule
                ],
                declarations: [
                    COMPONENTS
                ],
                exports: [
                    COMPONENTS
                ]
            }]
    }], null, null); })();
