// @zxing/ngx-scanner/esm2015/lib/zxing-scanner.module.js
import { CommonModule } from '@angular/common';
import { NgModule } from '@angular/core';
import { FormsModule } from '@angular/forms';
import { ZXingScannerComponent } from './zxing-scanner.component';
import * as ɵngcc0 from '@angular/core';
export class ZXingScannerModule {
}
ZXingScannerModule.ɵmod = ɵngcc0.ɵɵdefineNgModule({ type: ZXingScannerModule });
ZXingScannerModule.ɵinj = ɵngcc0.ɵɵdefineInjector({ factory: function ZXingScannerModule_Factory(t) { return new (t || ZXingScannerModule)(); }, imports: [[
            CommonModule,
            FormsModule
        ]] });
(function () { (typeof ngJitMode === "undefined" || ngJitMode) && ɵngcc0.ɵɵsetNgModuleScope(ZXingScannerModule, { declarations: function () { return [ZXingScannerComponent]; }, imports: function () { return [CommonModule,
        FormsModule]; }, exports: function () { return [ZXingScannerComponent]; } }); })();
(function () { (typeof ngDevMode === "undefined" || ngDevMode) && ɵngcc0.ɵsetClassMetadata(ZXingScannerModule, [{
        type: NgModule,
        args: [{
                imports: [
                    CommonModule,
                    FormsModule
                ],
                declarations: [ZXingScannerComponent],
                exports: [ZXingScannerComponent]
            }]
    }], null, null); })();
